confz/confz_source.py

```diff
@@ -37,7 +37,7 @@ class ConfZFileSource(ConfZSource):
 
 @dataclass
 class ConfZEnvSource(ConfZSource):
-    """Source config for environment variables. Environment variable names are transformed to
+    """Source config for environment variables and .env files. On loading of the source, the dotenv file values(if available) are merged with the environment, with environment always taking precedence in ase of name collusion. All loaded variable names are transformed to
```

Comment:
Would be good to respect max line length ;)

Reply:
Reflected.
As a potential enhancement - I had to turn off my python configuration as we rely on autoformatting via black + isort/flake/mypy, which was introducing too many irrelevant changes to the PR. Creating a lint target in pyproject or in an additional makefile would help contributers to always respect the code standard on PR.

Reply:
Great!
Yes, I was actually already planning to add black, mypy and pylint to the project, just did not find the time yet, but it really would make sense.